src/main/java/software/sigma/bu003/internship/coparts/client/technomir/entity/responce/StockPartFromTechnomir.java

```diff
@@ -0,0 +1,30 @@
+package software.sigma.bu003.internship.coparts.client.technomir.entity.responce;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+public class StockPartFromTechnomir {
+
+    private Long productId;
+    private String brand;
+    private String code;
+    private String descriptionRus;
+    private Integer quantity;
+    private Double price;
+    private String currency;
+    private String codePrinted;
+    private Double priceForRemote;
+
+    public void setPrice(String price) {
+        price = price.replace(",", "");
```

Comment:
move all logic from setter / getters - if you have a bug here it will be rally hard to find it, also you need to write unit tests for that, you can create converter method instead

Reply:
Api's answer has some bugs in some lines, this fixes them